keyboards/geminate60/via.json

```diff
@@ -0,0 +1,200 @@
+{

```

Comment:
This file doesn't belong in the QMK repo.  it needs to be PRed to the-via/keyboards